apps/nocodb/0.258.10/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   nocodb:
-    image: nocodb/nocodb:0.258.10
+    image: nocodb/nocodb:0.258.11
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
The provided code difference shows an update from version `0.258.10` to `0.258.11`. This involves changing the image tag from `nocodb/nocodb:0.258.10` to `nocodb/nocodb:0.258.11`, which is simply swapping minor versions without altering functionality or compatibility.

There are no apparent irregularities, potential issues, or optimization suggestions in this change since it involves updating a specific part of the configuration files used by Docker containers running NocoDB applications with different versions that might include bug fixes or performance improvements.